airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/DocumentationPanel.tsx

```diff
@@ -75,7 +75,7 @@ export const DocumentationPanel: React.FC = () => {
       {!docs.includes("<!DOCTYPE html>") ? (
         <DocumentationContent content={docs} rehypePlugins={urlReplacerPlugin} />
       ) : (
-        <DocumentationContent content={"Oops!  We do not yet have a Setup Guide for this connector."} />
+        <DocumentationContent content={"No Setup Guide found for this connector."} />
```

Comment:
@andyjih let me know what you think of this wording.  It is _very_ rare that a user would see this.  Currently only two connectors do not have docs, and @Amruta-Ranade is already on those.  I need to migrate this to i18n and make it full-height, but thought I'd get PM input on wording first.

<img width="1618" alt="Screen Shot 2022-05-12 at 11 00 37 AM" src="https://user-images.githubusercontent.com/63751206/168106050-255e3de2-6a6c-49c2-9b45-44a346a065fd.png">


An alternative would be showing https://docs.airbyte.com/quickstart/add-a-source but it doesn't seem very helpful for this scenario.

Reply:
@tealjulia That works for me, especially given how unlikely it is. Thanks!